support-frontend/assets/components/adFreeSectionC/adFreeSectionC.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react';
+
+// styles
+import './adFreeSectionC.scss';
+import '../../pages/digital-subscription-landing/components/digitalSubscriptionLanding.scss';
+
+
+const AdFreeSectionC = () => (
```

Comment:
What is the C in AdFreeSectionC?

Reply:
I'm not totally sure anymore! It might just be that the styling is different. It would be worth reviewing whether we could reuse the B version without massive style collisions through nesting.